### PR TITLE
Update calculated property error description

### PR DESCRIPTION
**Pull request by JonasDov**
Closes https://github.com/iTwin/imodel-native/issues/880

### CODE REVIEW

File: iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.h [hunk 1]
```diff
@@ -173,6 +173,17 @@ struct IECExpressionsCacheProvider
...
 
+//===================================================================================
+// @bsiclass
+//===================================================================================
+enum class ECExpressionEvaluationResult
+    {
+    EVALUATION_Success,
+    EVALUATION_ParseError,
+    EVALUATION_EvaluationError,
+    EVALUATION_InvalidECValueError
```
Comment (grigasp) [inline iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.h:184]: The `EVALUATION_` prefix is not needed for enum class

File: iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.h [hunk 2]
```diff
@@ -173,6 +173,17 @@ struct IECExpressionsCacheProvider
     ECExpressionsCache& Get(Utf8CP rulesetId) {return _Get(rulesetId);}
 };
 
+//===================================================================================
+// @bsiclass
+//===================================================================================
+enum class ECExpressionEvaluationResult
```
Comment (grigasp) [inline iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.h:179]: I'd probably rename this from "result" to "status".

**Review by grigasp — APPROVED** [review]

File: iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.h (hunk 1 above)
Comment (JonasDov) [reply iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.h]: Removed

File: iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.h (hunk 2 above)
Comment (JonasDov) [reply iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.h]: Renamed

**Pull request merged**